tools/PowerShell/Microsoft.WinGet.Client/src/Library/Get-LanguageData.ps1

```diff
@@ -0,0 +1,34 @@
+Function Get-LanguageData {
+    PARAM(
+        [Parameter(Position=0)] $Language = (Get-UICulture).Name
+    )
+
+    $languageData = $(
+        $hash = @{}
+
+        $(try {
+            # We have to trim the leading BOM for .NET's XML parser to correctly read Microsoft's own files - go figure
+            ([xml](((Invoke-WebRequest -Uri "https://raw.githubusercontent.com/microsoft/winget-cli/master/Localization/Resources/$Language/winget.resw" -ErrorAction Stop ).Content -replace "\uFEFF", ""))).root.data
```

Comment:
Having the build extract just the entries that are needed and shipping those subset .resw files with the module would be better.  Even having to download every language for the ~10 strings one needs would be smaller than downloading all of the strings for 1 language.

Reply:
Good idea!